src/drafts/SelectPanel2/SelectPanel.test.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import {ThemeProvider, ActionList} from '../../'
+import {render, RenderResult} from '@testing-library/react'
+import userEvent, {UserEvent} from '@testing-library/user-event'
```

Comment:
Oo UserEvent type, I've never used it before 🔥

Reply:
It's really nice!